### ISSUE TITLE
Strange CSS Import query (?wbr2mk) breaks webpack

### ISSUE THREAD

**Issue by JeremyBYU** [Flyer53/jsPanel3#8]
Hi! 
I was having some issues importing your projects css with webpack. I pinned down the issue in the css to the following: [Link](https://github.com/Flyer53/jsPanel3/blob/44a962cb926893f12c6abc45bd7fc2aa435ab49e/source/jquery.jspanel.css#L5-L6)

It seems to be this `?wbr2mk` format that you use when requesting fonts. I looked online but could not find any information about what this means.  I'm sure there is a reason for it, but was wondering if you could enlighten me.  After understanding, I will probably find a way to get weback to import your raw css.  Right now I have to manually delete it!

Thanks!

**Comment by Flyer53** [comment]
Hi Jeremy,
there is nothing to enlighten really ;) This <code>?wbr2mk</code> was created by the tool I used to create the font and also delivered the css and is absolutely unnecessary. I just didn't notice in time because I didn't have any problems with it. If you don't mind change the <code>@font-face</code> rule to the following:

``` css
@font-face {
  font-family: "jsglyph";
  src: url("fonts/jsglyph.eot");
  src: url("fonts/jsglyph.eot?#iefix") format("embedded-opentype"), url("fonts/jsglyph.ttf") format("truetype"), url("fonts/jsglyph.woff") format("woff"), url("fonts/jsglyph.svg#jsglyph") format("svg"); }
```

and it should work ... I hope ...

**Comment by Flyer53** [comment]
This issue is fixed, although I have to test the fix some more since I tested only on FF so far. If you want to test yourself you can download 3.0.1 beta
http://jspanel.de/downloads/jsPanel-3.0.1-beta.zip
This also includes the fix for #7

**Comment by JeremyBYU** [comment]
Wow, thanks for the speedy turn around! I have downloaded your source and copied it over into my `node_modules` folder.  Everything seems to working perfectly! 

Thanks so much for this!
